app/models/patient.rb

```diff
@@ -541,6 +542,9 @@ def send_assessment(force = false)
     # Do not allow messages to go to household members
     return unless responder_id == id
 
+    # Stop execution if in CI
+    return if Rails.env.test?
```

Comment:
Any reason to not put this at the very top? I will also say it is has been useful to be able to check that email assessments are sent when in the test environment even, which this change will prohibit.

Reply:
What exactly do you mean by test environment? I actually do not like having this check **at all** anywhere. We should strive to remove it completely. I did just notice it was never on the email send, which means things are even less consistent.

Reply:
I was referring to when `Rails.env.test` is true. Yeah the idea (I'm assuming) was that this check was on the contact methods that actually require Twilio being set up to avoid unnecessary errors there. Can likely be handled better though, agreed. I just wanted to note that when testing locally even (or even for running the tests that we should write on the CI), it's nice to have the emails work when Rails.env.test is true.

Reply:
I'm likely being silly and missing them, but what current tests depend on this actually sending an email?

Reply:
None right now - I was saying when we do write tests for this method (which we need to).

Reply:
I'll make an effort to remove it completely since this PR is now going to include 770.